journald/manifest.json

```
@@ -0,0 +1,35 @@
+{
+  "display_name": "journald",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "journald",
+  "metric_prefix": "journald.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Monitor your systemd-journald logs with Datadog.",
+  "guid": "845431ef-9092-4254-a188-138fc9273fa5",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "journald",
```

```suggestion
  "public_title": "Datadog-journald Integration",
```
To match other `manifest.json`

---

The template removes that now. The public name is used as the name of the tile, so the older convention is too wordy